samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -85,9 +85,50 @@ static struct bt_audio_stream_ops stream_ops = {
 	.sent = stream_sent_cb
 };
 
+static int setup_broadcast_source(struct bt_audio_broadcast_source **source)
+{
+	struct bt_audio_broadcast_source_stream_param
+		stream_params[CONFIG_BT_AUDIO_BROADCAST_SRC_STREAM_COUNT];
+	struct bt_audio_broadcast_source_subgroup_param
+		subgroup_param[CONFIG_BT_AUDIO_BROADCAST_SRC_SUBGROUP_COUNT];
+	struct bt_audio_broadcast_source_create_param create_param;
+	const size_t streams_per_subgroup = ARRAY_SIZE(stream_params) / ARRAY_SIZE(subgroup_param);
```

Comment:
This works when the stream count is the same per group. It can stay as is or maybe add a comment or check for clarification for devs picking up the sample.

Reply:
This was added to properly test multiple subsgroups with multiple streams in it. 

I think when we work towards https://github.com/zephyrproject-rtos/zephyr/issues/52347 this will be modified again, and this comment will be resolved by that